UCLA Radio/UCLA Radio/NotificationManager.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  NotificationManager.swift
+//  UCLA Radio
+//
+//  Created by Nathan Smith on 12/6/16.
+//  Copyright © 2016 ChrisLaganiere. All rights reserved.
+//
+
+import Foundation
+import UserNotifications
+
+class NotificationManager {
+    
+    static let sharedInstance = NotificationManager()
+    
+    /*
+    var notificationsAuthorized:Bool {
+        get {
+            if #available(iOS 10.0, *) {
+                let center = UNUserNotificationCenter.current()
+                center.getNotificationSettings() {_ in 
+                    
+                }
+            }
+        }
+    }*/
```

Comment:
Yes--sorry that was from something I was playing around with and never got around to deleting.